ui/src/views/DeferredQueue.vue

```diff
@@ -0,0 +1,629 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+<template>
+  <div>
+    <cv-grid fullWidth>
+      <cv-row>
+        <cv-column class="page-title">
+          <h2>
+            {{ $t("queue.title") }}
+            <cv-interactive-tooltip
+              alignment="start"
+              direction="right"
+              class="tooltip info mg-left-sm"
+            >
+              <template slot="trigger"></template>
+              <template slot="content">
+                <div v-html="$t('queue.queue_tooltip')"></div>
+              </template>
+            </cv-interactive-tooltip>
+          </h2>
+        </cv-column>
+      </cv-row>
+      <cv-row>
+        <cv-column>
+          <NsInlineNotification
+            kind="warning"
+            :title="core.$t('common.use_landscape_mode')"
+            :description="core.$t('common.use_landscape_mode_description')"
+            class="landscape-warning"
+          />
+        </cv-column>
+      </cv-row>
+      <cv-row class="toolbar">
+        <cv-column>
+          <template>
+            <NsButton
+              kind="secondary"
+              :icon="Restart20"
+              @click="listDeferredQueue"
+              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              >{{ $t("queue.reload_queue") }}
+            </NsButton>
+          </template>
+          <template v-if="deferred_queue.length">
+            <NsButton
+              kind="secondary"
+              :icon="Email20"
+              class="mg-left"
+              @click="setResendQueueAll"
+              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              >{{ $t("queue.resend_all") }}
+            </NsButton>
+            <NsButton
+              kind="danger"
+              class="mg-left"
+              :icon="TrashCan20"
+              @click="toggleDeleteQueueAll"
+              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              >{{ $t("queue.delete_all") }}
+            </NsButton>
+          </template>
+        </cv-column>
+      </cv-row>
+      <cv-row>
+        <cv-column>
+          <NsDataTable
```

Comment:
Let's add a `<cv-tile light>` between `<cv-column>` and `<NsDataTable>` to have a little more contrast on the page elements